config/initializers/assets.rb

```diff
@@ -9,9 +9,4 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in the app/assets
 # folder are already added.
-# Rails.application.config.assets.precompile += %w( admin.js admin.css )
-
-# Add Yarn node_modules folder to the asset load path.
-# Rails.application.config.assets.paths << Rails.root.join('node_modules')
-
-Rails.application.config.assets.paths << Rails.root.join("node_modules")
+Rails.application.config.assets.precompile += %w[test-dependencies.js]
```

Comment:
We should be able to remove this line of code as well, it was later removed as part of this commit - [Move to manifest.js for new sprockets](https://github.com/alphagov/smart-answers/pull/5881/commits/1961b28a5e30eea30c929b5d754cba08f3d2e1bd)